src/commands/skullboard.py

```diff
@@ -150,3 +465,73 @@ async def edit_or_send_skullboard_message(
+
+    @app_commands.command(name="about", description="Learn about the Skullboard")
+    async def about(self, interaction: Interaction):
+        skullboard_info = (
+            "The Skullboard is a fun way to track popular posts and active users in our community.\n\n"
+            "When a post receives a certain number of 💀 reactions, it gets added to the Skullboard.\n"
+            "Users earn 'skulls' for their popular posts, and these skulls contribute to their overall ranking.\n\n"
+            "Use `/skull stats rank` to see the current user rankings, and `/skull stats hof` to view the Hall of Fame!"
+        )
+        await interaction.response.send_message(skullboard_info)
+
+
+class SkullStatGroup(app_commands.Group):
+    db: SkullBoardDB = None
+
+    def __init__(self):
+        super().__init__(name="stats", description="Get stats about the skullboard")
+        if not SkullStatGroup.db:
+            SkullStatGroup.db = SkullBoardDB()
+
+    @app_commands.command(name="rank", description="Provides ranking of users")
+    async def rank(self, interaction: Interaction):
+        try:
+            rankings = await SkullStatGroup.db.get_user_rankings()
+            if not rankings:
+                await interaction.response.send_message(
+                    "No rankings available at the moment."
+                )
+                return
+            no_interact = AllowedMentions().none()
+
+            # Format the rankings into a more readable message
+            msg = "User Ranking by number of posts sent to the skullboard:\n"
+            for i, (user_id, frequency) in enumerate(rankings[:10], start=1):
+                msg += f"{i}. 💀 {frequency} : <@!{user_id}>\n"
+
+            await interaction.response.send_message(msg, allowed_mentions=no_interact)
+        except Exception as e:
+            await interaction.response.send_message(f"An error occurred: {str(e)}")
+
+    @app_commands.command(
+        name="hof", description="Hall of fame rankings for the top posts"
+    )
+    async def hof(self, interaction: Interaction):
+        try:
+            hof_entries = await self.db.get_HOF()
+            if not hof_entries:
+                await interaction.response.send_message("The Hall of Fame is empty.")
+                return
+            no_interact = AllowedMentions().none()
+            # Format the HoF entries into a more readable message
+            msg = "Hall of Fame:\n"
+            for i, (post_id, user_id, channel_id, day, frequency) in enumerate(
+                hof_entries, start=1
+            ):
+                # silent mention
+                msg += f"{i}. 💀 {frequency} : https://discord.com/channels/{self.db.guild_id}/{channel_id}/{post_id} from <@!{user_id}>\n"
+
+            await interaction.response.send_message(msg, allowed_mentions=no_interact)
+        except Exception as e:
+            await interaction.response.send_message(f"An error occurred: {str(e)}")
```

Comment:
You can send these as embeds, like the skull board posts itself, to make it look nicer.

Reply:
![hgfgfff](https://github.com/user-attachments/assets/1ebb6806-eba6-4151-8cef-257ed52c653f)